docs/design/admission_control_limit_range.md

```diff
@@ -0,0 +1,122 @@
+# Admission control plugin: LimitRanger
+
+## Background
+
+This document proposes a system for enforcing min/max limits per resource as part of admission control.
```

Comment:
No problem.  There are implementation PRs per each design, I can add user documentation to those assuming you were good with this outline.